tests/unit/test_column_noise.py

```diff
@@ -161,8 +159,12 @@ def test_leave_blank(dummy_dataset):
 
     # Check for expected noise level
     expected_noise = config[Keys.CELL_PROBABILITY]
-    actual_noise = len(newly_missing_idx) / len(orig_non_missing_idx)
-    is_close_wrapper(actual_noise, expected_noise, 0.02)
+    fuzzy_checker.fuzzy_assert_proportion(
```

Comment:
Does this tab-complete when you don't instantiate the class and go through this function yield instead?

Reply:
I'll add a suggestion to this PR.

Reply:
Huh, I had no idea type hints would extend to tab completion

Reply:
Might depend on your IDE! It works for me in VS Code 😃